invoice_comment_template/README.rst

```diff
@@ -0,0 +1,56 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+================
+Invoice Comments
+================
+
+Adds comments on invoices.
+The comments can be edited directly on the invoices or loaded from
+templates.
+
+Two positions are available for the comments:
+
+- before invoice lines
+- after invoice lines
+
+Usage
+=====
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/94/10.0
```

Comment:
change Odoo version from 10.0 to 11.0

Reply:
Or better, use the new README by fragments system and regenerate automatically the template.

Reply:
Yeah, I'll change it to the new README since I always forget about that!